<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyerTermsDialog</class>
 <widget class="QDialog" name="BuyerTermsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buyer</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="maxPiecePriceLabelTitle">
       <property name="text">
        <string>Maximum price per piece (...)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="maxPiecePriceLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="maxLockTimeLabel">
       <property name="text">
        <string>Maximum lock time (hours)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="minNumberOfSellersLabel">
       <property name="text">
        <string>Mininimum number of sellers</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="minNumberOfSellersLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="maxContractFeePrKbLabel">
       <property name="text">
        <string>Maximum contract fee (.../Kb)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="maxContractFeePrKbLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="totalLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Total funds required (...)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="totalValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="maxLockTimeLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="displayModeButton">
       <property name="text">
        <string>Simple</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minNumberOfSellersLineEdit</tabstop>
  <tabstop>maxContractFeePrKbLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
